apps/wallet-mobile/src/yoroi-wallets/cardano/api/utils.ts

```diff
@@ -61,8 +62,8 @@ export const toPolicyId = (tokenIdentifier: string) => {
   const tokenSubject = toTokenSubject(tokenIdentifier)
   return tokenSubject.slice(0, 56)
 }
-export const toAssetName = (tokenIdentifier: string) => {
-  return hexToUtf8(toAssetNameHex(tokenIdentifier))
+export const toDisplayAssetName = (tokenIdentifier: string) => {
+  return hexToUtf8(toUntaggedAssetNameHex(tokenIdentifier))
```

Comment:
it should drop the `hexToUtf8` cuz it's not following the protocol specs, it needs to consume --> `.asciiName ?? .hexName` directly from the `AssetNameUtils.`